src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterials.java

```diff
@@ -0,0 +1,34 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import com.kuljava.swiatwsi.world.Village;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "raw_materials")
+@Getter
+@Setter
```

Comment:
Why do You need setters?

Reply:
No I don't need

Reply:
So if You don't need it, You should remove them. YAGNI - principle for example. Another thing is that we want to keep our objects immutable as far as it is possible.